base/shared/src/main/scala/scalaz/typeclass/Semigroup.scala

```diff
@@ -1,6 +1,11 @@
 package scalaz
 package typeclass
 
+/**
+ * A Semigroup is a binary opertion such that:
+ * 1 - The result is in the same Set/Type
```

Comment:
This seems a little extraneous, as a quick glance at the type of `append` tells you exactly this. I'd opt for terseness and say

> A semigroup is a type equipped with an associative binary operation

Also, `opertion` is misspelled...